src/app/pushclient.js

```diff
@@ -1,86 +1,129 @@
-var request = require('request');
+/**
+ * You need to rename this file to PushClient.js
+ * Good OS are case-sensitive.
+ */
+
+const request = require('request');
 
 module.exports = class PushClient {
 
   constructor(pushConfig, retryOptions) {
-    if(!pushConfig.organizationId) throw new Error(`PushClient needs an organizationId in the pushConfig.`);
-    if(!pushConfig.sourceId) throw new Error(`PushClient needs an sourceId in the pushConfig.`);
-    if(!pushConfig.pushtoken) throw new Error(`PushClient needs an pushtoken in the pushConfig.`);
+    if(!pushConfig.organizationId) {
```

Comment:
I really prefer single lines here for readability, otherwise we add 6 lines for just validations of stuff most people are not concerned with while reading anyway.